### DISCUSSION TITLE
Database Log-In

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5446] [Q&A] [answered]
I know I'm missing something but following the install documents haven't worked as well as they should.
Powered by [AzuraCast](https://www.azuracast.com/) 
v0.16.1 Stable
Docker
PHP 8.1
This is where I started:
https://docs.azuracast.com/en/getting-started/requirements 
**my server** 
RAM 64Meg
Architecture: x86_64
CPU op-mode(s): 32-bit, 64-bit
CPU(s): 12
Vendor ID: AuthenticAMD
Model name: AMD Opteron(tm) Processor 6338P

I followed this install:
https://docs.azuracast.com/en/getting-started/installation/docker
Install was successful and I logged to the admin dashboard and set up 1 station and so far everything seems OK except for the missing directories and files in /var/azuracast
Next I install Let's Encrypt and still working fine even with the missing path
so I moved on, AzuraCast is working technically and the first station was broadcasting but when cloned that station I and renamed it I got the first error:
RuntimeException at /var/azuracast/www/src/Entity/Station.php L965: Media storage location not initialized.

I know that if I could log in to the database I could fix my issues but trying to log in to the database with the info in azuracast.env 
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
I'm getting that error because there isn't a mysqld.sock in the path

How do I log in to the database? mysql and maria commands don't work and checking the status 
systemctl status mariadb
Unit mariadb.service could not be found.
same error with MySQL

What am I missing?

**Comment by BusterNeece** [comment, marked as answer]
@440music This is all taking place inside Docker containers. Docker is a virtualized OS, basically (though far lighter weight than that), so all of our "internal" files, like the files in `/var/azuracast`, the DB, etc...are contained within the Docker ecosystem. This can be confusing at first, but once you get the hang of it, it's not too difficult.

Our utility script has some helpers to handle this separation:

```
./docker.sh bash
```

This will get you into a terminal _inside_ the running AzuraCast container, where all of the files will be exactly where AzuraCast sees them, so you can easily manipulate things as needed.

```
./docker.sh db
```

This will connect you to the database inside the container using the credentials in the environment file.

**Reply by ghost** [reply]
I removed 1 row from the database and the station that had been duplicated started working!
